dsdl_compiler/libcanard_dsdl_compiler/LICENSE

```diff
@@ -0,0 +1,23 @@
+pyratemp was written by Roland Koebler <rk at simple-is-better dot org>,
```

Comment:
The file should be renamed to indicate that it applies to Pyratemp only and not to the whole compiler.